website/profile/views.py

```diff
@@ -822,3 +822,13 @@ def redirect_to_twitter(twitter_handle):
         })
 
     return redirect(user.url)
+
+def view_institution(**kwargs):
+    from website.models import Institution
```

Comment:
Is it necessary to do this import here? If so, please add a comment explaining why (e.g. `# Avoid circular import`).